src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  deleteAnimal(animal) {
+    Animal.alive = Animal.alive.filter((item) => item !== animal);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  setBite() {
+    if (this.hidden === false) {
```

Comment:
```suggestion
    if (!this.hidden) {
```

Reply:
Consider implementing a check of the instance of the animal

Reply:
It doesn't work, because hidden might be undefined